chapter/3/message-passing.md

```diff
@@ -1,11 +1,250 @@
 ---
 layout: page
-title:  "Message Passing"
-by: "Joe Schmoe and Mary Jane"
+title:  "Message Passing and the Actor Model"
+by: "Nathaniel Dempkowski"
 ---
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. {% cite Uniqueness --file message-passing %}
+# Introduction
 
-## References
+Message passing programming models have essentially been discussed since the beginning of distributed computing and as a result message passing can be taken to mean a lot of things. If you look up a broad definition on Wikipedia, it includes things like RPC, CSP, and MPI. In practice when people talk about message passing today they mostly mean the actor model.
```

Comment:
I would add a sentence or two that recognizes that the concurrency model of the pi-calculus is another important inspiration for practical message passing systems. For example, Go's channels are based on the idea of first-class communication channels from the pi-calculus. Then, Clojure's core.async library is based on CSP. You can then say that you focus on the actor model given its ubiquity and generality.